eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -87,10 +87,10 @@ function Get-TocMapping {
         }
         $orderServiceMapping[$artifact] = @($serviceName, $displayName)
     }
-    return $orderServiceMapping                   
+    return $orderServiceMapping
 }
 
-function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang) {
+function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang, [String]$campaignId = "UA-62780441-46") {
```

Comment:
This "default" is the `general` campaign Id.

Reply:
Does this value need to be protected?

Reply:
No. It's stamped into every website that we serve. No way to keep it private.